xla/service/gpu/thunk.h

```diff
@@ -271,13 +277,15 @@ class Thunk {
   struct ExecuteParams {
     // Constructs execute parameters from an executable run options. Return
     // error if run options are misconfigured.
-    static ExecuteParams Create(const ServiceExecutableRunOptions& run_options,
-                                const BufferAllocations& buffer_allocations,
-                                se::Stream* stream,
-                                se::Stream* command_buffer_trace_stream,
-                                absl::Span<se::Stream* const> async_streams,
-                                CollectiveExecuteParams* collective_params,
-                                CollectiveCliques* collective_cliques);
+    static ExecuteParams Create(
+        const ServiceExecutableRunOptions& run_options,
+        const BufferAllocations& buffer_allocations, se::Stream* stream,
+        se::Stream* command_buffer_trace_stream,
+        absl::Span<se::Stream* const> async_streams,
+        CollectiveExecuteParams* collective_params,
+        CollectiveCliques* collective_cliques,
+        absl::flat_hash_map<ExecutionStreamId, se::Stream*>
+            additional_compute_streams = {});
```

Comment:
```
Thunk {
  using ExecutionStreams = absl::flat_hash_map<...>;
}
```

and `ExecutionStreams execution_streams` arg here, to save typing absl::flat_hash_map every time

Reply:
good point.